components/director/internal/graphql/scalars/helpers.go

```diff
@@ -0,0 +1,52 @@
+package scalars
```

Comment:
I don't like the idea of `helpers.go` file. Maybe we could move it to `pkg` as exported functions. However, we need a good name for that package 😄  Maybe `pkg/scalar`?

Reply:
moved to pkg/scalar/